languages/python/bitwarden_sdk/bitwarden_client.py

```diff
@@ -2,7 +2,16 @@
 from typing import Any, List, Optional
 from uuid import UUID
 import bitwarden_py
-from .schemas import ClientSettings, Command, ResponseForSecretIdentifiersResponse, ResponseForSecretResponse, ResponseForSecretsDeleteResponse, SecretCreateRequest, SecretGetRequest, SecretIdentifiersRequest, SecretIdentifiersResponse, SecretPutRequest, SecretResponse, SecretsCommand, SecretsDeleteRequest, SecretsDeleteResponse, AccessTokenLoginRequest, AccessTokenLoginResponse, ResponseForAccessTokenLoginResponse, ResponseForProjectResponse, ProjectsCommand, ProjectCreateRequest, ProjectGetRequest, ProjectPutRequest, ProjectsListRequest, ResponseForProjectsResponse, ResponseForProjectsDeleteResponse, ProjectsDeleteRequest
+
+from .schemas import ClientSettings, Command, ResponseForSecretIdentifiersResponse, ResponseForSecretResponse, \
```

Comment:
for long lines it's better to use parentheses, like so:
```suggestion
from .schemas import (ClientSettings, Command, ResponseForSecretIdentifiersResponse, 
    ResponseForSecretResponse, ResponseForSecretsResponse
```
you can also do each import in new line.

Reply:
Fixed in 86bb6de6d02ac1835cd4e511615b50fe36975fda.